dipy/align/__init__.py

```diff
@@ -21,7 +23,8 @@ def __init__(self, **kwds):
 DEBUG : print as much information as possible to isolate the cause of a bug.
 """
 
-
-
-
-
+from dipy.align._public import (syn_registration, register_dwi_to_template,
+                                write_mapping, read_mapping, resample,
+                                c_of_mass, translation, rigid, affine,
+                                affine_registration, register_series,
+                                register_dwi_series, streamline_registration)
```

Comment:
It would be good if you could add `__all__ = ['syn_registration', 'register_dwi_to_template', etc..]`

Reply:
Done in 78d835550